client/index.html

```diff
@@ -83,17 +83,17 @@
           <h1>¡Gracias! Solo falta un paso más:</h1>
           <div class="sr-instructions">
             <p>
-              Instrucciones de pago en OXXO:
-              <p>1. Entregue el código de al cajero en cualquier OXXO para que lo escanee.</p>
-              <p>2. Proporcione el pago en efectivo al cajero.</p>
+              Instrucciones para realizar el pago en OXXO:
```

Comment:
Let's make sure the text here is consistent with the text in the integration guide: https://site-admin.stripe.com/docs/payments/oxxo#oxxo-payment-instructions. If we get these translations approved, let's change the integration guide to match

Reply:
Looks like we've incorporated the translation updates from https://docs.google.com/document/d/1gsvYhRS1csIVzsg3p1YjA-zWS5nV6Fa4G5M1BO_jf-4/edit?disco=AAAAGevGr9w&ts=5eb09271&usp_dm=true
Thanks!

Reply:
All suggestions made by belen were made